server/routes/caseAdmin/assessment.ts

```diff
@@ -11,14 +12,45 @@ export default class AssessmentRoutes {
       res.locals.agent,
       req.params.prisonNumber,
     )
-    const { status } = assessmentSummary
 
+    const assessmentContactsResponse = await this.caseAdminCaseloadService.getCurrentAssessmentContactDetails(
+      req?.middleware?.clientToken,
+      res.locals.agent,
+      req.params.prisonNumber,
+    )
+
+    const { status } = assessmentSummary
     res.render('pages/caseAdmin/assessment', {
       assessmentSummary: {
         ...assessmentSummary,
         tasks: assessmentSummary.tasks.PRISON_CA,
+        contactList: this.createContactData(assessmentContactsResponse.contacts),
         formsToShow: getStatusFormsMap[status],
       },
     })
   }
+
+  private createContactData(contacts: ContactResponse[]) {
+    const probationCom = this.createContact(contacts, 'PROBATION_COM')
+    const prisonDm = this.createContact(contacts, 'PRISON_DM')
+    const prisonCa = this.createContact(contacts, 'PRISON_CA')
+    return [prisonCa, probationCom, prisonDm]
+  }
+
+  private createContact(
+    contacts: ContactResponse[],
+    role: 'PRISON_CA' | 'PRISON_DM' | 'PROBATION_COM',
+  ): ContactResponse {
+    const contact = contacts ? contacts.find(item => item.userRole === role) : null
```

Comment:
so presumably this isn't needed anymore: `contacts ? contacts.find(...`

if it was needed you can do: `contacts?.find(...`

Reply:
good to now some of my js/ts is rusty